Split SDKs into separate pages (#1280)
stellar/stellar-docs: docs/tools/sdks/contract-sdks.mdx

---
title: "Contract SDKs"
sidebar_position: 10
---

> Contract SDKs are used to build and interact with smart contracts on the Stellar network.

Reviewer: ```suggestion
Contract SDKs are used to build smart contracts that will be deployed to the Stellar network.
```

:::note

For Client and XDR SDKs, visit this [page](./library.mdx).

:::

The Rust SDK is maintained by SDF. The AssemblyScript SDK is maintained by dedicated community developers. All SDKs are open-source; file a GitHub issue or pull request in the specific SDK repository if you have questions or suggestions.

Each SDK has its own source code and documentation. Learn how to use a specific SDK by referring to the documentation.

> ## Soroban Rust SDK

Reviewer: I think we should add a section below this, `## Soroban Rust Token SDK`.

It is available here: https://docs.rs/soroban-token-sdk, and is also maintained by SDF and contains utilities for building tokens and can be used in addition to the soroban-sdk.

We could add this in a follow up.

cc @sisuresh

[Rust SDK](https://github.com/stellar/rs-soroban-sdk) | [Docs](https://docs.rs/soroban-sdk)

> Also known as the Smart Contract SDK. The `soroban-sdk` Rust crate contains the Soroban Rust SDK for building smart contracts on Stellar.

Reviewer: ```suggestion
The `soroban-sdk` Rust crate contains the Soroban Rust SDK for building smart contracts for Stellar.
```
